tests/unit/test_configurator/test_configurator_logs_exporter.py

```diff
@@ -115,61 +115,7 @@ def test_configure_logs_exporter_otel_none_sw_false(
         mock_blprocessor.assert_not_called()
         mock_logginghandler.assert_not_called()
 
-    def test_configure_logs_exporter_otel_true_sw_false(
```

Comment:
Removing this test coverage because logs/metrics exporters are always configured now based on `OTEL_(LOGS|METRICS)_EXPORTER` etc, with defaults set by Distro (e.g. [this PR](https://github.com/solarwinds/apm-python/pull/499/files)).